### PR TITLE
Comments Redesign: Add CommentAuthorMoreInfo

### PR DESCRIPTION
**Pull request by Copons**
Follow up to #19184

Add the new `CommentAuthorMoreInfo` block.

screen shot 2017-10-27 at 15 21 29

screen shot 2017-10-27 at 15 20 00

### Testing instructions

Add this code in `CommentList` to display a list of comments using the new design:

```jsx
import Comment from 'my-sites/comments/comment';
//...
{ ! isLoading &&
	map( commentsPage, commentId => (
		<Comment
			commentId={ commentId }
			key={ `comment-${ siteId }-${ commentId }` }
			isBulkMode={ isBulkEdit }
			isSelected={ this.isCommentSelected( commentId ) }
		/>
	) ) }
```

### Diagram

screen shot 2017-10-23 at 21 11 14

### CODE REVIEW

**Comment by kwight** [comment]
@Copons are there more details about testing? Looks like we also need to replace the `CommentDetail` import with `import Comment from 'my-sites/comments/comment';`, but this still just got me a list of all anonymous users, so I've only seen the one version of user info popover:

screen shot 2017-10-27 at 9 50 39 am

**Comment by Copons** [comment]
@kwight Ah yeah, if you replace the `CommentDetail` map, you lose the comments data, because I've not added `QueryComment` into the new component, which so currently lives off `CommentDetail` data.

Just add the `Comment` map right above the `CommentDetail` one, also handy to check the differences.

**Comment by gwwar** [comment]
Looks like we probably don't want to display user info for pingbacks or did we change that on this iteration?

screen shot 2017-10-27 at 3 21 43 pm

File: client/my-sites/comments/comment/comment-author-more-info.jsx [hunk 1]
```diff
@@ -3,33 +3,253 @@
...
+							</Button>
+						</div>
+					) }
+
+					{ ! authorEmail && (
+						<div className="comment__author-more-info-element">
+							<div>
+								{ translate(
+									// eslint-disable-next-line max-len
+									"Anonymous messages can't be blocked individually, but you can update your {{a}}settings{{/a}} to only allow comments from registered users.",
```
Comment (gwwar) [inline client/my-sites/comments/comment/comment-author-more-info.jsx]: Did we run into trouble with translate doing a string concat here?

Comment (Copons) [reply client/my-sites/comments/comment/comment-author-more-info.jsx]: It never occurred to me that `translate` accepted concatenated strings!

**Comment by Copons** [comment]
> Looks like we probably don't want to display user info for pingbacks

@gwwar Good catch, I totally missed this.

**Comment by Copons** [comment]
Concatenated the long string and hidden the More Info from non-`comment` comments.

**Pull request merged**